src/patterns/pages/error-status-pages/examples/timed-out-error-not-signed-in/index.njk

```diff
@@ -0,0 +1,18 @@
+---
+layout: none
+---
+{% extends "layout/_template.njk" %}
+
+{% set pageConfig = {
+    "title": "Service name",
+    "footer": {
+    }
+} %}
+
+{% block main %}
+
+    <h1>Sorry, you need to sign in again</h1>
+
```

Comment:
There should still be a bullet list on this example:
```
This is because you have either:
- been inactive for 45 minutes and your session has timed out to protect your information
- followed a link to the middle of a request for a new access code
```

Reply:
Done

Reply:
Done

Reply:
Need to import the list macro